docs/automation-rules.rst

```diff
@@ -51,6 +51,8 @@ When a rule matches a new version, the specified action is performed on that ver
 Currently, the following actions are available:
 
 - **Activate version**: Activates and builds the version.
+- **Hide version**: Hides the version. If the version is not active, activates it and builds the version.
+  See more in :ref:`guides/hiding-a-version:Hide a Version and Keep its Docs Online`.
```

Comment:
Not sure about linking to a guide here, I was thinking linking to https://github.com/readthedocs/readthedocs.org/blob/migrate-null-ranks/docs/versions.rst#version-states